smd_thumbnail.php

```diff
@@ -1454,21 +1453,31 @@ function smd_thumb_switch_active_done(data) {
                 $link_del = join_qs($qs).a.'step=smd_thumb_profile_delete'.a.'smd_thumb_name='.$row['name'];
                 $btnEdt = '<a href="'.$link_edt.'">[' . gTxt('edit') . ']</a>';
                 $btnDel = '<a href="'.$link_del.'" onclick="return confirm(\''.gTxt('smd_thumb_delete_confirm', array("{name}" => $row['name'])).'\');">[' . gTxt('delete') . ']</a>';
-                $btnDel = href('×', array(
-                    'event'          => 'image',
-                    'step'           => 'smd_thumb_profile_delete',
-                    'smd_thumb_name' => $row['name'],
-                    '_txp_token'     => form_token(),
-                    'page'           => $page,
-                    'sort'           => $sort,
-                    'dir'            => $dir,
-                    'crit'           => $crit,
-                    'search_method'  => $search_method,
-                ), array(
-                    'class'       => 'dlink destroy ui-icon ui-icon-close',
-                    'title'       => gTxt('delete'),
-                    'data-verify' => gTxt('smd_thumb_delete_confirm', array("{name}" => $row['name'])),
-                ));
+                $btnDel = tag(
```

Comment:
Why an anchor inside a button? I can't get the delete to work now. Delete button just does... nothing.